docs/superpowers/plans/2026-07-02-openapi-codemode-provider.md

```
@@ -0,0 +1,1009 @@
+```rust
+pub async fn dispatch_openapi_call(registry: &OpenApiRegistry, client: &reqwest::Client,
+    label: &str, operation_id: &str, params: serde_json::Value) -> Result<serde_json::Value, OpenApiError>
+{
+    let handle = registry.operation(label, operation_id)?; // unknown label/op → structured error
+    let out = crate::http::execute_operation(client, handle, params).await?;
+    tracing::info!(service="openapi", action=operation_id, label=%label,
+        host=%handle.base_url.host_str().unwrap_or_default(), status="ok",
+        "openapi dispatch complete"); // NO body, NO url-with-query, NO auth
+    Ok(out)
+}
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)